src/components/NoteDialog/NoteDialog.test.tsx

```diff
@@ -0,0 +1,121 @@
+import {render} from "@testing-library/react";
+import configureStore from "redux-mock-store";
+import {Provider} from "react-redux";
+import {wrapWithTestBackend} from "react-dnd-test-utils";
+import NoteDialog from "./NoteDialog";
+
+const mockStore = configureStore();
+
+const createNoteDialog = (text: string, authorId: string, showAuthors: boolean) => {
+  const initialState = {
+    board: {
+      data: {
+        columns: [{id: "test_column", name: "test_header", hidden: false}],
+      },
+    },
+    notes: [],
+    users: {
+      admins: [
+        {
+          id: "jkKqOUgt3hEDvl7CWcBokVOGs6AzINon",
+          displayName: "Kinetic Kobold",
+          admin: true,
+          createdAt: "2021-08-11T10:45:41.640Z",
+          updatedAt: "2021-08-11T10:52:21.558Z",
+          online: true,
+        },
+      ],
+      basic: [],
+      all: [],
+    },
+  };
+  const store = mockStore(initialState);
+  const [NoteDialogContext] = wrapWithTestBackend(NoteDialog);
+  return (
+    <Provider store={store}>
+      <NoteDialogContext
+        noteId="0"
+        isAdmin
+        text={text}
+        authorId={authorId}
+        columnName=""
+        columnColor=""
+        show
+        activeVoting
+        authorName=""
+        showAuthors={showAuthors}
+        votes={[
+          {
+            id: "test-id",
+            board: "test-board",
+            note: "0",
+            user: "test-user",
+          },
+        ]}
+        childrenNotes={[
+          {id: "1", columnId: "TestID", text: "", author: "", parentId: "0", dirty: true, authorName: "", votes: []},
```

Comment:
Above (line 13) you set the column id to "test_column" but here you use "TestID"

Reply:
Correct. Should be fixed now.